client/ayon_core/hosts/houdini/plugins/create/create_alembic_camera.py

```diff
@@ -26,9 +27,14 @@ def create(self, subset_name, instance_data, pre_create_data):
             pre_create_data)  # type: CreatedInstance
 
         instance_node = hou.node(instance.get("instance_node"))
+
+        filepath = "{root}/{subset}/{subset}.abc".format(
+            root=hou.text.expandString(self.staging_dir),
+            subset=subset_name
+        )
```

Comment:
This is really much better as channel references to the `subset` parm on the node itself, so that if the user would toggle the target asset/subset etc. down the line that.

```suggestion
        filepath = '$HIP/`chs("subset")`/`chs("subset")`.abc'.format(
            root=hou.text.expandString(self.staging_dir),
            subset=subset_name
        )
```

Also, I really wonder why we've decided to not keep these paths relative to `$HIP` since that's a very normal/regular Houdini workflow but instead making these by expanded strings.

Reply:
> Also, I really wonder why we've decided to not keep these paths relative to `$HIP`

Do paths relative to `$HIP` raise a conflict on the farm ?

Reply:
Correct - that would be the case - if that file would be moved over, yes. However, I'd actually resolve that in the area where you're moving things over instead of forcing it in the workfile. So preferably the workfile publish might "bake" it - but leaving the local scene as is.